modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CreateZoneCommand.java

```diff
@@ -121,6 +127,10 @@ private CatalogZoneDescriptor descriptor(int objectId) {
                 ? dataStorageParams
                 : DataStorageParams.builder().engine(DEFAULT_STORAGE_ENGINE).dataRegion(DEFAULT_DATA_REGION).build();
 
+        List<StorageProfileParams> storageProfileParams0 = storageProfileParams != null
```

Comment:
I think we need to avoid the zones without storage profiles with appropriate error. Instead, for the default-based short path we should use the appropriate "default" storage profile for:
- default zone, which is created by  our scripts
- the value of table storage profile, if it is not specified manually in DDL

Reply:
@alievmirza so, do we still need this default fallback to dummy* here?

Reply:
Thanks, forgot to remove